podaac/subsetter/subset.py

```diff
@@ -322,8 +322,8 @@ def get_coord_variable_names(dataset):
         Tuple of strings, where the first element is the lat coordinate
         name and the second element is the lon coordinate name
     """
-    possible_lat_coord_names = ['lat', 'latitude', 'y']
-    possible_lon_coord_names = ['lon', 'longitude', 'x']
+    possible_lat_coord_names = ['lat', 'latitude', 'y', 'Latitude']
```

Comment:
For this and time below, can we just check against var.lower() instead of adding this extra code? Like

```python
def var_is_coord(var_name, possible_coord_names):
        var_name = var_name.strip(GROUP_DELIM).split(GROUP_DELIM)[-1]
        return var_name.lower() in possible_coord_names
```

and 

```python
for var_name in list(dataset.data_vars.keys()):
        if 'time' in var_name.lower() and dataset[var_name].squeeze().dims[0] in lat_var.squeeze().dims:
            return var_name
```

Reply:
Ah yes